src/main/java/io/supertokens/storage/mysql/Start.java

```diff
@@ -310,14 +314,14 @@ public void createNewSession(String sessionHandle, String userId, String refresh
         }
     }
 
-    @Override
-    public void deleteSessionsOfUser(String userId) throws StorageQueryException {
-        try {
-            SessionQueries.deleteSessionsOfUser(this, userId);
-        } catch (SQLException e) {
-            throw new StorageQueryException(e);
-        }
-    }
+//    @Override
+//    public void deleteSessionsOfUser(String userId) throws StorageQueryException {
+//        try {
+//            SessionQueries.deleteSessionsOfUser(this, userId);
+//        } catch (SQLException e) {
+//            throw new StorageQueryException(e);
+//        }
+//    }
```

Comment:
Why is this commented out? Either merge the latest release into this branch, or then remove the comment.

Reply:
+ Revert the deleted functions. I think the issue is that the plugin interface repo doesn't have the newest things merged.

Reply:
I am on the latest passwordless branch of interface, after the recent updates I had to remove some interface implementations. I removed the comment